Source/Entities/MOSParticle.cpp

```diff
@@ -20,6 +20,7 @@ MOSParticle::~MOSParticle() {
 void MOSParticle::Clear() {
 	m_Atom = nullptr;
 	m_SpriteAnimMode = OVERLIFETIME;
+	m_PostEffectEnabled = true;
```

Comment:
please add a comment describing why this is being done (backwards compat reasons)